blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -185,6 +185,7 @@ sealed abstract class BlazeClientBuilder[F[_]] private (
   def resource: Resource[F, Client[F]] =
     scheduler.flatMap { scheduler =>
       verifyAllTimeoutsAccuracy(scheduler)
+      verifyTimeoutRelations()
```

Comment:
These impure calls are in pure position.

Reply:
Indeed these methods write some logs. I don't see any examples of logging being handled as an effect in http4s code base, so that's the pattern I followed. Do you want me to wrap them in `F`?

Reply:
`F.delay(logger` will show a few examples.  log4cats is only a dependency of the ember modules.